src/example_dapp_one_chain_battle_game/game_assets/external/operations.rell

```diff
@@ -61,6 +62,18 @@ operation register_and_mint_asset_hero(
     1,
     [account.id]
   );
+
+  for(attribute in attributes) {
```

Comment:
instead of this I propose to call in this operation
  originals_light_interfaces.require_interface_compatibility(
    attributes,
    originals_light_interfaces.interface @ { .id == game_assets.config.hero_interface_id }
  );

and later just call originals_light.register_and_mint_asset